app/bundles/EmailBundle/Form/Type/ConfigType.php

```diff
@@ -74,6 +74,24 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ]
         );
 
+        $builder->add(
+            'preferences_center_success_message',
+            TextareaType::class,
+            [
+                'label'      => 'mautic.email.config.preferences_center_success_message',
+                'label_attr' => ['class' => 'control-label'],
+                'attr'       => [
+                    'class'   => 'form-control',
+                ],
+                'required'   => false,
+                'data'       => (array_key_exists('preferences_center_success_message', $options['data']) && !empty($options['data']['preferences_center_success_message']))
+                    ? $options['data']['preferences_center_success_message']
+                    : $this->translator->trans(
+                        'mautic.email.preferences_center_success_message.text'
+                    ),
```

Comment:
This could be simplified to
```suggestion
                'data'       => $options['data']['preferences_center_success_message'] ?? $this->translator->trans('mautic.email.preferences_center_success_message.text'),
```
But it will set the default message only if the config does not exist or is null (similar to `isset()`). Not sure if we need to cover also an empty string here.